src/main/java/com/raisetech/raisetechtask/mapper/UserMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.raisetech.raisetechtask.mapper;
+
+import com.raisetech.raisetechtask.entity.User;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Select;
+
+import java.util.List;
+
+@Mapper
+public interface UserMapper {
+
+    @Select("SELECT * FROM names")
+    List<User> findAll();
+
+    @Select("SELECT * FROM names WHERE id = #{id}")
+    List<User> findById(int id);
```

Comment:
```suggestion
    Optional<User> findById(int id);
```
このようにしたいです。
理由は、このメソッドはPK検索なので検索結果は1件取得できる or 検索結果がないの2点になるはずです。
その点を考慮すると戻り値の型はListでなくOptionalにするのがよいかとおもいます。

Optionalとはなんだ？となるかと思います。
こちらについてはぜひ調べてみてください。

こちらもPR別で対応しちゃいましょうか。